Dockerfile.centos7

```
@@ -0,0 +1,31 @@
+FROM centos:7
+ENV HOME /
+RUN yum update -y
+RUN yum install -y epel-release
+RUN yum install -y rpm-build make gcc-c++ tar gzip cmake3 pkgconfig systemd-devel cyrus-sasl-devel zlib-devel flex bison unzip openssl-devel libgcrypt-devel
+
+ARG FLB_PREFIX
+ARG FLB_VERSION
+ARG FLB_RELEASE
+
+ENV FLB_TARBALL http://github.com/fluent/fluent-bit/archive/$FLB_PREFIX$FLB_VERSION.zip
+
+RUN cd /tmp && \
+    curl -sSL -o "/tmp/fluent-bit-${FLB_VERSION}.zip" ${FLB_TARBALL} && \
+    unzip "fluent-bit-$FLB_VERSION.zip"
+WORKDIR "/tmp/fluent-bit-${FLB_VERSION}/build"
+
+# override package information
+# because it is not the official package td-agent.
+RUN cat ../CMakeLists.txt | \
+    perl -pe "s/\\bset\\(CPACK_PACKAGE_NAME\\s+.*\\)\$/set(CPACK_PACKAGE_NAME \"fluent-bit\")/g" | \
+    perl -pe "s/\\bset\\(CPACK_PACKAGE_RELEASE\\s+.*\\)\$/set(CPACK_PACKAGE_RELEASE \"$FLB_RELEASE.el7\")/g" | \
+    perl -pe "s/\\bset\\(CPACK_PACKAGE_CONTACT\\s+.*\\)\$/set(CPACK_PACKAGE_CONTACT \"Manabu Seki <github@zitaku.net>\")/g" | \
+    perl -pe "s/\\bset\\(CPACK_PACKAGE_VENDOR\\s+.*\\)\$/set(CPACK_PACKAGE_VENDOR \"Manabu Seki\")/g" | \
```

Can you set my name? because the rpm package will be signed by my GPG key.

```suggestion
    perl -pe "s/\\bset\\(CPACK_PACKAGE_CONTACT\\s+.*\\)\$/set(CPACK_PACKAGE_CONTACT \"Ichinose Shogo <shogo82148@gmail.com>\")/g" | \
    perl -pe "s/\\bset\\(CPACK_PACKAGE_VENDOR\\s+.*\\)\$/set(CPACK_PACKAGE_VENDOR \"Ichinose Shogo\")/g" | \
```